README.md

```diff
@@ -18,3 +18,18 @@ Vi bruker `h2` som database under utvikling. For å se, endre og slette kandidat
 ```
 JDBC URL: jdbc:h2:mem:testdb
 ```
+
+
+# Henvendelser
+
+## For Nav-ansatte
+* Dette Git-repositoriet eies av [Team inkludering i Produktområde arbeidsgiver](https://navno.sharepoint.com/sites/intranett-prosjekter-og-utvikling/SitePages/Produktomr%C3%A5de-arbeidsgiver.aspx).
+* Slack-kanaler:
+  * [#inkludering-utvikling](https://nav-it.slack.com/archives/CQZU35J6A)
+  * [#arbeidsgiver-utvikling](https://nav-it.slack.com/archives/CD4MES6BB)
+  * [#arbeidsgiver-general](https://nav-it.slack.com/archives/CCM649PDH)
+
+## For folk utenfor Nav
```

Comment:
Vi har skrudd av "Github issues" på dette og andre repos, så det øverste punktet er misvisende. Skal vi skru dette på igjen?

Reply:
Ref. samtale: Siden vi ikke kommer på noen bedre måte for folk utenfor Nav  å kontakte oss på enabler vi issues igjen.